src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table (name = "countries")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and providing a `clone` method is not a common practice for entity classes in Hibernate. Entities are usually managed by the Hibernate session and their lifecycle is tied to the database context, so cloning them can lead to unexpected behavior and should be avoided unless there is a specific requirement for it.